ios/brave-ios/App/Client.xcodeproj/xcshareddata/xcschemes/Debug (No Core).xcscheme

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1250"
+   version = "1.7">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "YES">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "BraveCore Pre-actions"
+               scriptText = "sh &quot;${PROJECT_DIR}/../scripts/xcode_scheme_preaction.sh&quot; --only_update_symlink&#10;">
```

Comment:
I wonder if this can be done via some XCode project/scheme flag that everyone can just toggle YES/NO at any point in any scheme, and you can access it as an env variable in `xcode_scheme_preaction.sh`.